assets/js/modules/search-console/components/dashboard-details/DashboardDetailsWidgetKeywordsTable.js

```diff
@@ -42,9 +43,16 @@ const DashboardDetailsWidgetKeywordsTable = () => {
 		resource_id: propertyID,
 		num_of_days: getCurrentDateRangeDayCount(),
 	};
+	const isDomainProperty = useSelect( ( select ) => select( STORE_NAME ).isDomainProperty() );
+	const referenceSiteURL = useSelect( ( select ) => {
+		return untrailingslashit( select( CORE_SITE ).getReferenceSiteURL() );
+	} );
 	if ( url ) {
 		footerCTALinkArgs.page = `!${ url }`;
 	}
+	if ( isDomainProperty && referenceSiteURL ) {
```

Comment:
This needs to be an `else if` for the above, otherwise it would override the more specific URL for the current entity, which is against the ACs: If there is a specific URL to show results for, there's no point in limiting to a certain URL prefix anyway since the URL would have that by definition and be more specific.